### PR TITLE
Translation added

### PR DESCRIPTION
**Pull request by Gallimathias**
I have added English and German translations to the Address Service. Others can of course be added. This enables people to view the new translation function. ;)

### CODE REVIEW

File: samples/GetAddressService/Pages/Addresses.al [hunk 1]
```diff
@@ -14,16 +14,28 @@ page 50103 Addresses
...
         {
             repeater(General)
             {
-                field(Address;Address) { ApplicationArea = All; }
-                field(Locality;Locality) { ApplicationArea = All; }
-                field("Town/City";"Town/City") { ApplicationArea = All; }
-                field(County;County) { ApplicationArea = All; }
+                Caption = 'General';
+
+                field(Address;Address) {
```
Comment (StanislawStempin) [inline samples/GetAddressService/Pages/Addresses.al]: The formatting style that we would like is:
```
field(Address;Address)
{
    Caption = ...
    ApplicationArea = ...
}
```

Comment (Gallimathias) [reply samples/GetAddressService/Pages/Addresses.al]: I just turned this into a commit [4aa9cd0e](https://github.com/Microsoft/AL/pull/968/commits/4aa9cd0eed7df2f32f9439fc458ed6ae3d32627d)

**Comment by Gallimathias** [comment]
@StanislawStempin I have commit the format changes and removed the binary files. I also added a gitignore for the. app files.

**Comment by StanislawStempin** [comment]
The .app files are still part of the pull request. 
.gitignore will prevent future additions of .app files but you need to remove these ones manually.

**Comment by Gallimathias** [comment]
@StanislawStempin Ok, now I have removed all binary files. On last commit i have Overlooked both files in the .alpackage folder and deleted only the packed .app file in the Parent folder ^^.

Now it should fit.

**Pull request merged**